assets/js/base/components/quantity-selector/index.js

```diff
@@ -5,8 +5,9 @@ import { __, sprintf } from '@wordpress/i18n';
 import { speak } from '@wordpress/a11y';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import { useRef, useState, useEffect, useCallback } from '@wordpress/element';
+import { useCallback } from '@wordpress/element';
 import { DOWN, UP } from '@wordpress/keycodes';
+import { isUndefined } from 'lodash';
```

Comment:
I'd prefer not importing functions from lodash that are easy enough to do in native js (unless you're also importing something else from lodash).  I'm not going to be sticky on this point though if you want to keep it.